packages/ckeditor5-email/src/emailintegrationutils.ts

```diff
@@ -0,0 +1,196 @@
+		 * It is encouraged to remove the plugin from the editor configuration to maintain compatibility.
+		 * While it is possible to disable this warning using the `email.warnings.suppress`
+		 * configuration option, it is strongly discouraged.
+		 *
+		 * @error email-integration-unsupported-plugin
+		 */
+		this._logSuppressibleWarning( 'email-integration-unsupported-plugin', { pluginName } );
+	}
+
+	/**
+	 * Logs a warning about an unsupported color value used in the configuration.
+	 *
+	 * @internal
+	 */
+	public _validateConfigColorValue<const K extends string>( configPath: NeverIfUnknownConfigPath<K> ): void {
+		const colorConfig = this.editor.config.get( configPath ) as string | Array<ColorOption> | undefined;
+
+		if ( !colorConfig ) {
+			return;
+		}
+
+		const isArrayConfig = Array.isArray( colorConfig );
+		const colors = isArrayConfig ? colorConfig : [ colorConfig ];
+
+		for ( const [ index, item ] of colors.entries() ) {
+			const color = typeof item === 'string' ? item : item.color;
+
+			if ( isUnsupportedEmailColorValue( color ) ) {
+				/**
+				 * The specified color value uses a format not supported in email clients. This affects various color
+				 * settings like fonts, backgrounds, borders, etc. Please use `rgb()` or `#RRGGBB` format instead.
+				 *
+				 * @error email-integration-unsupported-color-value
+				 */
+				this._logSuppressibleWarning( 'email-integration-unsupported-color-value', {
+					configPath: isArrayConfig ? `${ configPath }[${ index }]` : configPath,
+					color
+				} );
+			}
+		}
+	}
+
+	/**
+	 * Logs a warning about an unsupported color format used in the configuration.
+	 *
+	 * @internal
+	 */
+	public _validateConfigColorFormat<const K extends string>( configPath: NeverIfUnknownConfigPath<K> ): void {
+		const format = this.editor.config.get( configPath ) as string | undefined;
+
+		if ( !format ) {
+			return;
+		}
+
+		if ( isUnsupportedEmailColorFormat( format ) ) {
+			/**
+			 * The color format specified in the editor configuration (e.g. for color pickers or other UI components)
+			 * is not supported in email clients. Please use `rgb()` or `#RRGGBB` format instead.
+			 *
+			 * @error email-integration-unsupported-color-format
+			 */
+			this._logSuppressibleWarning( 'email-integration-unsupported-color-format', { configPath, format } );
+		}
+	}
+
+	/**
+	 * Checks if the log with the given code should be suppressed.
+	 *
+	 * @internal
+	 */
+	private _isSuppressedLog( warningCode: string, data?: object ): boolean {
+		const editor = this.editor;
+		const config = editor.config.get( 'email.logs' )!;
+
+		if ( config.suppressAll ) {
+			return true;
+		}
+
+		if ( typeof config.suppress === 'function' && config.suppress( warningCode, data ) ) {
+			return true;
+		}
+
+		if ( Array.isArray( config.suppress ) && config.suppress.includes( warningCode ) ) {
+			return true;
+		}
+
+		return false;
+	}
+}
+
+const UNSUPPORTED_COLOR_FORMATS = [ 'hsl', 'hsla', 'hwb', 'lab', 'lch' ];
```

Comment:
I would extend this list a bit:

```suggestion
const UNSUPPORTED_COLOR_FORMATS = [ 'hsl', 'hsla', 'hwb', 'lab', 'lch', 'oklab', 'oklch' ];
```

I would also consider adding `color-mix` to above list 👉 https://developer.mozilla.org/en-US/docs/Web/CSS/color_value/color-mix

BTW, in error logs we suggest to use `rgb()` or `#RRGGBB` format - what about `rgba()`? Is it valid or should be added to the unsupported list? 🤔

Reply:
Most likely RGBA not work, I'll add it.